### PR TITLE
Add the functionality to view the plotted  spectrum in log scale

### LINKED ISSUE #624: log scale in plot
I was using Radis-app for my project today ; and found out I really need to have a log-scale option for the y-axis in Plots.

- is it possible to include it directly in the plotting library (is it Plotly?) ? (i.e. it would be a button embedded in the plot, and would not require to recompute the spectrum nor to add a new button ) 

- if not possible, I'd add a tick box "Logscale" above the "Calculate" button. If we end up having many plot options, we could group them under a same "Show plot options" tick box (later)

### PR DESCRIPTION
**Pull request by arunavabasu-03**
Fixes #624

https://user-images.githubusercontent.com/73842340/186729508-73fc3697-7701-4377-8fcb-1da44ddcfcbb.mp4

### CODE REVIEW

File: frontend/src/components/CalcSpectrumPlot.tsx [hunk 1]
```diff
@@ -81,6 +105,7 @@ const CalcSpectrumPlot_ = ({
               type: "linear",
               fixedrange: false,
             },
+            updatemenus: updatemenus,
```
Comment (suzil) [inline frontend/src/components/CalcSpectrumPlot.tsx:108]:
```suggestion
            updatemenus,
```
this isn't important but it's some syntactic sugar you can use if the property name equals the variable name

**Review by suzil — CHANGES_REQUESTED** [review]
The ylabel seems to have disappeared in this branch. It disappears if you click log and then back to linear.

Before:
Screen Shot 2022-08-25 at 1 27 51 PM
After:
Screen Shot 2022-08-25 at 1 28 06 PM

**Comment by suzil** [comment]
I was playing around a bit with the position of the buttons:
Screen Shot 2022-08-25 at 1 33 57 PM

Thoughts on this version @arunavabasu-03 @erwanp? It's a little more aligned with the left axis of the plot and raised up a little higher to not overlap with the grid lines.

Screen Shot 2022-08-25 at 1 33 27 PM

**Comment by suzil** [comment]
Alternatively here's a version in the bottom left:
Screen Shot 2022-08-25 at 1 36 31 PM
Screen Shot 2022-08-25 at 1 36 54 PM

**Comment by erwanp** [comment]
I like the bottom left version better !

**Review by suzil — APPROVED** [review]
Looks good

**Pull request merged**